src/components/node-selector.tsx

```
@@ -0,0 +1,197 @@
+        </SheetHeader>
+
+        <div>
+          {triggerNodes.map((nodeType) => {
+            const Icon = nodeType.icon;
+
+            return (
+              <div
+                key={nodeType.type}
+                className="hover:border-l-primary h-auto w-full cursor-pointer justify-start rounded-none border-l-2 border-transparent px-4 py-5"
+                onClick={() => handleNodeSelect(nodeType)}
+              >
```

Biome (2.4.14)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)